spec/factories/conversations.rb

```diff
@@ -7,5 +7,19 @@
       create :target, topic: initiator.targets.first.topic,
                       lat: initiator.targets.first.lat, lng: initiator.targets.first.lng
     end
+    unread { false }
```

Comment:
No need to add braces when the value is atomic (string, integer, boolean...)

Reply:
When removing this brackets I get:
`NoMethodError: undefined method 'unread' in 'conversation' factory`